site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
+        SEO Keywords: STS, endpoint services, temporary credentials, webIdentity, AssumeRole, WSO2, OPA, etcd, client grants
 
   - Minio Client:
       - Minio Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/mc/master/README.md
         Slug: minio-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | MinIO client quickstart guide
+        SEO Description: This document explains how to use the MinIO Client as a modern alternative to UNIX commands like ls, cat, cp, mirror, diff, find etc. For filesystems + Amazon S3 compatible services
+        SEO Keywords: minio client, UNIX, docker
         
 
       - Minio Client Complete Guide:
         Link: https://raw.githubusercontent.com/minio/mc/master/docs/minio-client-complete-guide.md
         Slug: minio-client-complete-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | The complete guide to the MinIO client
+        SEO Description: This document explains in detail how to use the MinIO Client as a modern alternative to UNIX commands like ls, cat, cp, mirror, diff, find etc. For filesystems + Amazon S3 compatible services
+        SEO Keywords: UNIX commands,
 
       - Minio Admin Complete Guide:
         Link: https://github.com/minio/mc/blob/master/docs/minio-admin-complete-guide.md
         Slug: minio-admin-complete-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | The complete guide to the MinIO Admin functionality
+        SEO Description: This document explains in detail how to use the MinIO Client's administrative functionality including service, info, user, policy, heal
+        SEO Keywords: service, info, policy, credential, config, heal, docker stable, docker edge
 
   - Minio SDKs:
       - JavaScript Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio-js/master/README.md
         Slug: javascript-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | JavaScript Client Quickstart Guide
+        SEO Description: This quickstart guide will show you how to install the client SDK and execute an example JavaScript program.
+        SEO Keywords: JavaScript, JavaScript API, npm install minio
 
       - JavaScript Client API Reference:
         Link: https://raw.githubusercontent.com/minio/minio-js/master/docs/API.md
         Slug: javascript-client-api-reference
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | JavaScript Client API Reference
+        SEO Description: This document contains the detailed example code for the JavaScript Client API
+        SEO Keywords: JavaScript Client, JavaScript API
```

Comment:
Same as my previous comment, should we add `MinIO JavaScript SDK` and `MinIO JavaScript Client` as keywords here, as those will be the main search phrases users will be using to access this page?

Reply:
yes

Reply:
add don't replace